src/components/MoneyRequestHeader.js

```diff
@@ -70,6 +71,8 @@ function MoneyRequestHeader(props) {
         setIsDeleteModalVisible(false);
     }, [parentReportAction, setIsDeleteModalVisible]);
 
+    const isScanning = !ReportActionsUtils.areAllRequestsBeingSmartScanned(parentReportAction);
```

Comment:
```suggestion
    const isScanning = ReportActionsUtils.areAllRequestsBeingSmartScanned(parentReportAction);
```

Can you please double check the usage of `areAllRequestsBeingSmartScanned` in other places